application/models/database/Database.php

```diff
@@ -0,0 +1,30 @@
+<?php 
+
+namespace Database;
+/**
+ * 
+ * @method __construct()
+ * It will  initialize connection with database
+ * 
+ */
+class Database
```

Comment:
In this page also please check indentaion like write `{` from same line where class name is written, add space before `{` and add extra line after end of the file